.github/workflows/push-release.yml

```diff
@@ -10,22 +10,20 @@ jobs:
     name: Build, test, and publish stable release
     if: "contains(github.event.head_commit.message, 'chore(release): publish')"
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [12.x]
+
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
           # checkout master explicitly as workflow triggered on a tag which results in checkout of detached HEAD
           ref: master
           # 0 indicates all history, needed for correct unstable version generation (commit count since last release)
           fetch-depth: 0
 
       - name: Setup nodejs
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
-          node-version: 10.x
+          node-version: 16.x
```

Comment:
Keep the pattern similar to other workflows with matrix? Unsure why this was different